NGCHM/WebContent/javascript/DetailHeatMapEvent.js

```diff
@@ -82,7 +82,7 @@ DEV.addEvents = function (paneId) {
 	    		} else if (distance > mapItem.latestPinchDistance){ // pinch inward
 				DEV.detailDataZoomIn(mapItem);
 	    		} else if (mapItem.latestPinchDistance > distance){ // pinch outward
-				DEV.detailDataZoomOut(e);
+				DEV.detailDataZoomOut(mapItem.chm);
```

Comment:
This looks suspicious.  detailDataZoomIn takes  "mapItem", while ZoomOut takes "mapItem.chm", while within detailDataZoomOut mapItem seems to be something different than mapItem here.

Reply:
They've been defined like that for a while.  We should fix it.  (Unless there's a compelling reason not to, but I doubt it.)